assets/src/scss/layout/_block-classes.scss

```diff
@@ -1,26 +1,79 @@
 .wp-block-query-pagination {
+  a,
+  .page-numbers {
+    color: $black;
+    padding: $sp-1 $sp-1x;
+    user-select: none;
+    border: 1px solid transparent;
+    border-radius: 4px;
+  }
+
   a {
-    color: black;
-    font-weight: bold;
+    font-weight: 500;
+
+    &:hover {
+      text-decoration: none;
+      background-color: $grey-10;
+      transition-property: background-color, border-color;
+      transition-duration: 250ms;
+      transition-timing-function: linear;
+    }
   }
 
   font-family: $roboto;
   display: flex;
   flex-direction: row;
-  justify-content: center;
+  justify-content: space-between;
   width: 100%;
-  margin-bottom: 16px;
+  margin-bottom: $sp-2;
+
+  @include large-and-up {
+    justify-content: center;
+  }
+}
+
+.wp-block-query-pagination-previous,
+.wp-block-query-pagination-next {
+  margin: 0 !important;
+}
+
+.wp-block-query-pagination-previous:before,
+.wp-block-query-pagination-next:after {
+  content: "";
+  height: 12px;
+  width: 8px;
+  display: inline-block;
+  mask-image: url("../../images/chevron.svg");
+  mask-repeat: no-repeat;
+  mask-size: contain;
+  mask-position: center;
+  background-repeat: no-repeat;
+  background-color: currentColor;
+  color: $black;
 }
 
-.page-numbers {
-  &:not(:last-child) {
-    margin-inline-end: 8px;
+.wp-block-query-pagination-previous {
+  &:before {
+    margin-right: 6px;
```

Comment:
This should be `margin-inline-end` to be compatible with RTL sites

Reply:
This is not necessary because of [this](https://github.com/greenpeace/planet4-master-theme/pull/1759/files#diff-8ab58c5a79269045226346866d479d279b91b44ca462bde4e8fd9d982e22a979R45-R47). It's already changed the direction on RTL sites.

Reply:
The RTL version is broken at the moment though, see this [general comment](https://github.com/greenpeace/planet4-master-theme/pull/1759#pullrequestreview-1055200635) that I left!

Reply:
> The RTL version looks a bit broken:
> 
> <img alt="Screenshot 2022-07-29 at 10 36 49" width="525" src="https://user-images.githubusercontent.com/6949075/181719545-bd8e4bb1-5ff1-4fd1-9d57-4d1a8ae49df7.png">

I assumed you were testing before the instance had been updated 🤔 

https://user-images.githubusercontent.com/77975803/181752997-37e033ef-75c5-450c-a17b-dad1b2230aa8.mov